helm/net-exporter/values.yaml

```diff
@@ -0,0 +1,22 @@
+name: net-exporter
+namespace: monitoring
```

Comment:
Damn this has the same namespace problem as cert-exporter.

What about we change the value to `.exporter.namespace` and then set it to `kube-system` in the default app catalog?

Reply:
+1000 this should not end up in the `monitoring` namespace in tenant clusters. That will be very confusing.

Reply:
should it belong to `kube-system` or `giantswarm` in TC? I'm confusing as well.

Reply:
`kube-system` same as it is now. `giantswarm` namespace is for chart-operator, tiler and related resources.

Reply:
See https://github.com/giantswarm/cert-exporter/pull/44#discussion_r337879908 

For tenant clusters we create a configmap that sets the namespace to `kube-system`.

In the control plane we use the `monitoring` namespace that is the default value. We need the equivalent of this logic for the new chart.